docs/vdrtools-cli/vdrtools-cli-identity-tutorial.md

```diff
@@ -0,0 +1,369 @@
+
+  
+
+Use this command:
+
+  
+
+```
+cheqd-pool add alias=cheqd-pool-testnet rpc_address=https://rpc.testnet.cheqd.network/ chain_id=cheqd-testnet-4
+```
+
+  
+
+Example:
+
+  
+
+```
+cheqd-wallet:indy> cheqd-pool add alias=cheqd-pool-testnet rpc_address=https://rpc.testnet.cheqd.network/ chain_id=cheqd-testnet-4
+Pool "cheqd-pool-testnet" has been created "{"alias":"cheqd-pool-testnet","rpc_address":"https://rpc.testnet.cheqd.network/","chain_id":"cheqd-testnet-4"}"
+```
+
+  
+
+#### 4\. Open a connection to the testnet: 
+
+  
+
+Next we need to setup a direct connection between the users machine and the pool on testnet.
+
+  
+
+Use this command:
+
+  
+
+```
+cheqd-pool open alias=cheqd-pool-testnet
+```
+
+  
+
+Example:
+
+  
+
+```
+cheqd-wallet:indy> cheqd-pool open alias=cheqd-pool-testnet
+Pool "cheqd-pool-testnet" has been opened
+```
+
+  
+
+#### 5\. Generate a DID:
+
+  
+
+Now the connection has been made, we're ready to create a DID. In this case we'll generate a DID first locally. This is because the current functionality of VDR tools only allows for the creation of a unique identifier as opposed to a DID.
+
+  
+
+To do this we'll use need to use a 3 step process using 2 different commands
+
+  
+
+1.  First use `did new`
+2.  Then copy the newly generated DID
+3.  Finally, paste your new DID into the allocated space in the templated command `did new did=did:cheqd:testnet:<copied-DID>`
+
+  
+
+Example:
+
+  
+
+```
+cheqd_pool(cheqd-pool-testnet):cheqd-wallet:indy> did new
+Did "X9cNs1g3nMvQ77j7E8rPRr" has been created with "~4zHub2htDfEuV7wCsNcEgg" verkey
+
+cheqd_pool(cheqd-pool-testnet):cheqd-wallet:indy> did new did=did:cheqd:testnet:X9cNs1g3nMvQ77j7E8rPRr
+Did "did:cheqd:testnet:X9cNs1g3nMvQ77j7E8rPRr" has been created with "7db7n4CNBbYFRcYEjoacMzSbVEqF2ToSovtvzBoovH5B" verkey
+```
+
+  
+
+#### 6\. Generate/restore cosmos signature:
+
+  
+
+Now we'll need to restore the Cosmos keys from the mnemonic. The mnemonic required here was provided during node operator account creation.
+
+  
+
+2-steps command:
+
+  
+
+*   `cheqd-keys add alias=cheqd-testnet-key mnemonic`
+*   paste your mnemonic after suggestion `Enter value for mnemonic:`
+
+  
+
+Example:
+
+  
+
+```
+cheqd_pool(cheqd-pool-testnet):cheqd-wallet:indy> cheqd-keys add alias=cheqd-testnet-key mnemonic
+Enter value for mnemonic:
+
+Key for account: cheqd1waszljnwadcuynjg6wc9rhhw8ega20nc97muc6
+has been restored from mnemonic
+```
+
+  
+
+#### 7\. Send previously created DID: 
+
+  
+
+We can now use the VDR tools to send our newly created DID using the following command:
+
+  
+
+```
+cheqd-ledger create-did did=<DID_from_step_5> max_coin=<value> max_gas=<value> denom=ncheq memo=memo key_alias=cheqd-testnet-key
+```
+
+  
+
+Note: You only need to setup `max_gas` and `max_coin` parameters, according to your financial policy. Defaults are `max_gas=200000 max_coin=5000000`
```

Comment:
Makes no sense to specify both max gas and max coin, since fees = gas * gas price. So isn't just max_coin enough?

Reply:
But for now there are required parameters unfortunately and we have to setup both of them... We can send a PR to Evernym's gitlab in future.